tests/element_handle_test.go

```diff
@@ -375,6 +375,8 @@ func TestElementHandleScreenshot(t *testing.T) {
 	elem, err := p.Query("div")
 	require.NoError(t, err)
 
+	// TODO: Use a mock instead of a LocalFilePersister when there's no need to
+	//       persist files.
```

Comment:
Do we have a TODO for this somewhere on Github? Maybe add a task to the issue.

Reply:
Thanks for reminding me. I've added a `mockPersister` in 145648579f8c73fe46a8de4e329c52656f9fe271

Reply:
Nice ❤️